### PR TITLE
added support for increase paid storage in local forging package

### PR DESCRIPTION
**Pull request by dsawali**
#1766 
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: packages/taquito-local-forging/src/schema/operation.ts [hunk 1]
```diff
@@ -126,6 +126,16 @@ export const TxRollupSubmitBatchSchema = {
...
 };
 
+export const IncreasePaidStorageSchema = {
+  source: CODEC.PKH,
+  fee: CODEC.ZARITH,
+  counter: CODEC.ZARITH,
+  gas_limit: CODEC.ZARITH,
+  storage_limit: CODEC.ZARITH,
+  amount: CODEC.ZARITH,
+  destination: CODEC.ADDRESS,
```
Comment (roxaneletourneau) [inline packages/taquito-local-forging/src/schema/operation.ts:136]: The ADDRESS codec can encode implicit and smart contract addresses. Destination, in this case, should be a smart contract. Should we have a dedicated smartContractAddressEncoder?

**Review by roxaneletourneau — COMMENTED** [review]
small suggestion to consider, lgtm otherwise :)

File: packages/taquito-local-forging/src/schema/operation.ts (hunk 1 above)
Comment (dsawali) [reply packages/taquito-local-forging/src/schema/operation.ts:136]: that's a good point, let's add that

Comment (dsawali) [reply packages/taquito-local-forging/src/schema/operation.ts]: @roxaneletourneau I've added the new encoder/decoder

Comment (dsawali) [reply packages/taquito-local-forging/src/schema/operation.ts]: @roxaneletourneau oops let me add some unit tests for the new encoder

**Review by roxaneletourneau — APPROVED** [review]

**Pull request merged**